src/Driver/Pdo/AbstractPdoConnection.php

```diff
@@ -50,8 +51,11 @@ abstract class AbstractPdoConnection extends AbstractConnection implements PdoCo
     protected QuoterInterface|null $quoter = null;
     protected SchemaInterface|null $schema = null;
 
-    public function __construct(protected PdoDriverInterface $driver, protected SchemaCache $schemaCache)
-    {
+    public function __construct(
+        protected PdoDriverInterface $driver,
+        protected SchemaCache $schemaCache,
+        protected ColumnFactoryInterface|null $columnFactory = null,
```

Comment:
This must be used in DB implementations, right? Currently is not used.

Reply:
This is an option for user defined column factory
https://github.com/yiisoft/db/pull/941/files#diff-05f0a5783bcd92ea3d34bbe0c21bb3b2407837f6aa1efd441c87e5e08436dce3R292